terraform/P3/6-create-cloud-front.tf

```diff
@@ -0,0 +1,53 @@
+resource "aws_cloudfront_origin_access_identity" "tf_cloudfront_oai" {
+  comment = "tf_cloudfront"
```

Comment:
ça sert à quoi ça ?

Reply:
Amazon CloudFront Origin Access Identity (OAI) is a virtual user identity used for sharing private content via CloudFront. It is used to give the CF distribution permission to fetch a private object from the origin server, such as an S3 bucket. To work with Origin Access Identities, navigate to Clouds > AWS Global > CF Origin Access Identities. The CloudFront Origin Access Identities page lists all OAIs that were created by the RightScale account

Reply:
En gros c'est un identity check